challenge_2/ruby/selectivealso/methods.rb

```diff
@@ -0,0 +1,12 @@
+class Array
```

Comment:
I'm not a big fan of overriding core classes. While rails does this it creates a confusing environment where it's difficult to tell what came from where (unless you use `method(:foo).source_location`). What do you think about using a refinement instead? Or another alternative would be to create a different object that you instantiate with an array that returns the desired value.